packages/flutter/lib/src/material/drawer.dart

```diff
@@ -220,6 +221,10 @@ class DrawerController extends StatefulWidget {
   /// {@endtemplate}
   final DragStartBehavior dragStartBehavior;
 
+  /// The color to use for the scrim that obscures primary content while a drawer is open.
+  /// By default, the color is [Colors.black54]
```

Comment:
Being nitpicky but traditionally there is a line break between the opening sentence and the rest of the comment.

Reply:
done